frontend/src/features/whats-new/FeatureUpdateList.ts

```diff
@@ -13,10 +13,11 @@ export interface FeatureUpdate {
   image?: FeatureUpdateImage
 }
 
+// TODO: Confirm the actual features which will be showcased in the What's New section
 export const FeatureUpdateList: FeatureUpdate[] = [
   {
     id: 1,
-    date: '17 July',
+    date: '17 June 2022',
```

Comment:
since this is now an object instead of JSON, should we make this into a `Date` instead? So we do not need to perform any conversion when using it. 

To also enforce SG timezone, we can include GMT information:
```ts
date: new Date('17 Jun 2022 GMT+8')
```

Reply:
Yeap I think that makes sense. Initially, I didn't think of making it into Date type because it's only use was to display the date as a String for each feature update item. But since I am using it for sorting, it would be better if I just stored it as a date and use date-fns to format it into string when displaying it within the drawer.